### PR TITLE
Dockerise fetch

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Dockerise fetch raw data and move test contacts to pipeline_config.json

### CODE REVIEW

File: docker-run.sh [hunk 1]
```diff
@@ -100,30 +100,31 @@ fi
 docker start -a -i "$container"
 
 # Copy the output data back out of the container
-#echo "Copying traced data"
-#mkdir -p "$(dirname "$OUTPUT_JSON")"
-#docker cp "$container:/data/output.json" "$OUTPUT_JSON"
+echo "Copying out traced data..."
```
Comment (as2388) [inline docker-run.sh:103]: Might be even better to say "Copying traced data from '$container:/data/output.json' to '$OUTPUT_JSON'"

File: Pipfile [hunk 2]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 CoreDataModules = {editable = true,ref = "v0.9.1",git = "https://www.github.com/AfricasVoices/CoreDataModules"}
 PipelineInfrastructure = {editable = true,ref = "08cfb07be4c80656924bceaeeaef15a21ddb7a72", git = "https://www.github.com/AfricasVoices/PipelineInfrastructure"}
+RapidProTools = {editable = true,git = "https://www.github.com/AfricasVoices/RapidProTools",ref = "dfaa9c2284671e65a08662a77eb819ca5b1d7fa7"}
```
Comment (as2388) [inline Pipfile:9]: Can you switch to a released version (v0.2.6)

File: Pipfile [hunk 3]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 CoreDataModules = {editable = true,ref = "v0.9.1",git = "https://www.github.com/AfricasVoices/CoreDataModules"}
 PipelineInfrastructure = {editable = true,ref = "08cfb07be4c80656924bceaeeaef15a21ddb7a72", git = "https://www.github.com/AfricasVoices/PipelineInfrastructure"}
```
Comment (as2388) [inline Pipfile:8]: Same here (v0.0.1)

File: fetch_raw_data.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    for survey in SURVEYS:
+        output_file_path = f"{raw_data_dir}/{survey}.json"
+        print(f"Exporting survey '{survey}' to '{output_file_path}'...")
+
+        flow_id = rapid_pro.get_flow_id(survey)
+        raw_runs = rapid_pro.get_raw_runs_for_flow_id(flow_id)
+        raw_contacts = rapid_pro.update_raw_contacts_with_latest_modified(raw_contacts)
+        traced_runs = rapid_pro.convert_runs_to_traced_data(
+            user, raw_runs, raw_contacts, phone_number_uuid_table, rapid_pro_test_contact_uuids)
+        traced_runs = rapid_pro.coalesce_traced_runs_by_key(user, traced_runs, "avf_phone_id")
```
Comment (as2388) [inline fetch_raw_data.py:104]: (fyi when you switch to a released version of RapidProTools this function needs to move to the pipeline (see https://github.com/AfricasVoices/Project-ADSS/blob/7a582ab0cb7f0d2d03d5ec9819cfc79d81304fe5/generate_outputs.py#L114))

**Review by as2388 — APPROVED** [review]

File: docker-run.sh (hunk 1 above)
Comment (IsaackMwenda) [reply docker-run.sh:103]: Update thanks!

File: Pipfile (hunk 2 above)
Comment (IsaackMwenda) [reply Pipfile:9]: To fix in a future pr

File: Pipfile (hunk 3 above)
Comment (IsaackMwenda) [reply Pipfile:8]: Fixed https://github.com/AfricasVoices/Project-ICRAF/tree/run-pipeline-script

**Pull request merged**